deployment/RS01SUM2_Deploy.csv

```diff
@@ -1,13 +1,19 @@
-TN-313,,,,RS01SUM2-MJ01B-06-MASSPA101,2,1,2016-07-28T20:03:00,2017-08-10T23:00:00,ATAPL-65244-020-0024,,ATAPL-58325-00002,44.56996,-125.14686,,774,774,
-RR1714,,,,RS01SUM2-MJ01B-05-CAMDSB103,3,1,2017-08-12T01:00:00,,ATAPL-65244-020-0024,,ATAPL-58317-00010,44.56999,-125.14671,,774,774,
-RR1714,,,,RS01SUM2-MJ01B-06-MASSPA101,3,1,2017-08-11T23:00:00,,ATAPL-65244-020-0024,,ATAPL-58325-00002,44.56996,-125.14686,,774,774,
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
+TN-326,,,,RS01SUM2-MJ01B-00-FLOBNC101,1,1,2015-07-14T18:50:06,2016-07-22T00:00:00,ATAPL-65244-020-0024,,ATAPL-58334-00004,44.57,-125.14681,,775,775,
+TN-326,,,,RS01SUM2-MJ01B-00-FLOBNM101,1,1,2015-07-14T19:43:27,2016-07-22T00:00:00,ATAPL-65244-020-0024,,ATAPL-70113-00004,44.57,-125.1468,,774,774,
+TN-313,,,,RS01SUM2-MJ01B-05-CAMDSB103,1,1,2014-08-31T19:17:45,2016-07-22T00:00:00,ATAPL-65244-020-0024,,ATAPL-58317-00004,44.56999,-125.14671,,774,774,
+TN-313,,,,RS01SUM2-MJ01B-06-MASSPA101,1,1,2014-09-02T22:11:47,2016-07-22T00:00:00,ATAPL-65244-020-0024,,ATAPL-58325-00001,44.56996,-125.14686,,774,774,
+TN-313,,,,RS01SUM2-MJ01B-12-ADCPSK101,1,1,2014-08-29T07:24:46,2018-06-25T18:30:00,ATAPL-65244-020-0024,,ATAPL-58419-00002,44.56984,-125.14682,,773,773,
+TN-326,,,,RS01SUM2-MJ01B-00-FLOBNC101,2,1,2016-07-23T00:00:00,,ATAPL-65244-020-0024,,ATAPL-58334-00002,44.57,-125.14681,,775,775,
+TN-326,,,,RS01SUM2-MJ01B-00-FLOBNM101,2,1,2016-07-23T00:00:00,,ATAPL-65244-020-0024,,ATAPL-70113-00002,44.57,-125.1468,,774,774,
+TN-326,,,,RS01SUM2-MJ01B-00-OSMOIA101,2,1,2015-07-09T12:32:00,,ATAPL-65244-020-0024,,ATAPL-58333-00004,45.93367,-130.01369,,774,774,
+TN-313,,,,RS01SUM2-MJ01B-05-CAMDSB103,2,1,2016-07-19T20:14:00,2017-08-10T23:00:00,ATAPL-65244-020-0024,,ATAPL-58317-00010,44.56999,-125.14671,,774,774,
+TN-313,,,,RS01SUM2-MJ01B-06-MASSPA101,2,1,2016-07-28T20:03:00,2017-08-10T23:00:00,ATAPL-65244-020-0024,,ATAPL-58325-00002,44.56996,-125.14686,,774,774,
+RR1714,,,,RS01SUM2-MJ01B-05-CAMDSB103,3,1,2017-08-12T01:00:00,2018-06-25T18:30:00,ATAPL-65244-020-0024,,ATAPL-58317-00010,44.56999,-125.14671,,774,774,
+RR1714,,,,RS01SUM2-MJ01B-06-MASSPA101,3,1,2017-08-11T23:00:00,2017-08-10T23:00:00,ATAPL-65244-020-0024,,ATAPL-58325-00002,44.56996,-125.14686,,774,774,
+RR1809,,,,RS01SUM2-MJ01B-05-CAMDSB103,4,1,2018-06-28T22:00:00,,ATAPL-65244-020-0024,,ATAPL-58325-00003,44.56996,-125.14686,,774,774,
+RR1809,,,,RS01SUM2-MJ01B-06-MASSPA101,4,1,2018-06-28T22:00:00,,ATAPL-65244-020-0024,,ATAPL-58317-00010,44.56999,-125.14671,,774,774,
+RR1809,,,,RS01SUM2-MJ01B-12-ADCPSK101,4,1,2018-06-26T16:30:00,,ATAPL-65244-020-0024,,ATAPL-58419-00003,44.56984,-125.14682,,774,774,
+RR1809,,,,RS01SUM2-MJ01B-11-CTDPFA110,1,1,2018-06-28T20:00:00,,ATAPL-65244-020-0024,,PIRSN-CTDPFA-10001,44.57,-125.14671,,774,774,

```

Comment:
CTDPFA110, QNTSRA101, OVRSRA101 are all new instruments and will need to be added to the vocabulary listing